### PR TITLE
CLOUDP-126564: Move post packaging tests to evergreen generate task

### PR DESCRIPTION
**Pull request by fmenezes**
## Proposed changes

Move post packaging tests to evergreen generate task

_Jira ticket:_ CLOUDP-126564

Closes #[issue number]

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

**Comment by fmenezes** [comment]
Meta generated test will be red until atlascli 1.1.6 is released

**Review by andreaangiolillo — DISMISSED** [review]
LGTM

File: tools/genevergreen/main.go [hunk 1]
```diff
@@ -97,13 +106,66 @@ func generateRepoTasks(toolName string) *shrub.Configuration {
...
+
+func generatePostPkgMetaTasks(c *shrub.Configuration, toolName string) {
+	if toolName != atlascli {
+		return
+	}
+
+	v := &shrub.Variant{
+		BuildName:        fmt.Sprintf("pkg_smoke_tests_docker_meta_%v_generated", toolName),
+		BuildDisplayName: fmt.Sprintf("Generated post packaging smoke tests (Meta / %v)", toolName),
+		DistroRunOn:      []string{"ubuntu1804-small"},
```
Comment (gssbzn) [inline tools/genevergreen/main.go:144]: [non-blocking] can we make `ubuntu1804-small` a const and make sure we use it everywhere so when we need to update it is in a single place?
something like
```go
const runOn = "ubuntu1804-small"
```

**Review by gssbzn — DISMISSED** [review]
LGTM if LGTEVG

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**